docs/notebook_tutorial/handling_structued_data.ipynb

```diff
@@ -364,7 +365,7 @@
    "metadata": {},
    "source": [
     "## Build Coverage Index\n",
-    "`DataPack.get()` is commonly used to retrieve entries from a datapack. In some cases, we are only interested in getting entries from a specific range. `DataPack.get()` allows users to set `range_annotation` which controls the search area of the sub-types. If `DataPack.get()` is called frequently with queries related to the `range_annotation`, you may consider building the coverage index regarding the related entry types. Users can call `DataPack.build_coverage_for(context_type, covered_type)` in order to create a mapping between a pair of entry types and target entries that are covered in ranges specified by outer entries.\n",
+    "`DataPack.get()` is commonly used to retrieve entries from a data pack. In some cases, we are only interested in getting entries from a specific range. `DataPack.get()` allows users to set `range_annotation`, which controls the search area of the sub-types. If `DataPack.get()` is called frequently with queries related to the `range_annotation`, you may consider building the coverage index regarding the related entry types. Users can call `DataPack.build_coverage_for(context_type, covered_type)` to create a mapping between a pair of entry types and target entries covered in ranges specified by outer entries.\n",
```

Comment:
Recently we agree to unify the reference to DataPack. Let's always use the format `DataPack`, i.e. two words together, since this will make it more unique and searchable online.